util/kvdb-rocksdb/src/lib.rs

```diff
@@ -536,7 +547,7 @@ impl Database {
 						.expect("iterator params are valid; qed"));
 
 				Some(DatabaseIterator {
-					iter: iter,
+					iter: interleave_ordered(Vec::new(), iter),
```

Comment:
it should use [`std::iter::empty`](https://doc.rust-lang.org/std/iter/fn.empty.html) instead of `Vec::new`

Reply:
nvm, I see that types would not match